package/MDAnalysis/coordinates/GRO.py

```diff
@@ -326,7 +326,7 @@ def __init__(self, filename, convert_units=None, n_atoms=None, **kwargs):
                w.write(u.atoms)
 
         """
-        self.filename = util.filename(filename, ext='gro')
+        self.filename = util.filename(filename, ext='gro', keep = True)
```

Comment:
PEP8; no space around `=` in args.

Reply:
Are we also restricting the number of characters per line to 80/79 as per pep8?

Reply:
See the [Style Guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide): yes, we try to keep within 79 characters. (Not all our code rigorously adheres to it, but that's what we're striving for)